draft-ietf-core-corr-clar.md

```diff
@@ -51,6 +55,10 @@ informative:
     date: false
   RFC9146:
     # Connection Identifiers for DTLS 1.2. That keeps the session/epoch and enables to change the ip-address/port, if the matching is relaxed from the ip-endpoints.
+  COAP-RESOURCE:
+    target: https://libcoap.net/doc/reference/4.3.0/man_coap_resource.html
```

Comment:
I would have gone for https://libcoap.net/doc/reference/4.3.5/man_coap_resource.html, but maybe the later information is too verbose.